pkg/vulnloader/nvdloader/manual.go

```diff
@@ -297,4 +297,134 @@ var manuallyEnrichedVulns = map[string]*schema.NVDCVEFeedJSON10DefCVEItem{
 		LastModifiedDate: "2022-03-16T00:00Z",
 		PublishedDate:    "2022-03-16T00:00Z",
 	},
+	"CVE-2022-22963": {
+		CVE: &schema.CVEJSON40{
+			CVEDataMeta: &schema.CVEJSON40CVEDataMeta{
+				ID: "CVE-2022-22963",
+			},
+			DataFormat:  "MITRE",
+			DataType:    "CVE",
+			DataVersion: "4.0",
+			Description: &schema.CVEJSON40Description{
+				DescriptionData: []*schema.CVEJSON40LangString{
+					{
+						Lang:  "en",
+						Value: `In Spring Cloud Function versions 3.1.6, 3.2.2 and older unsupported versions, when using routing functionality it is possible for a user to provide a specially crafted SpEL as a routing-expression that may result in remote code execution and access to local resources.`,
+					},
+				},
+			},
+			References: &schema.CVEJSON40References{
+				ReferenceData: []*schema.CVEJSON40Reference{
+					{
+						Name: "https://tanzu.vmware.com/security/cve-2022-22963",
+					},
+				},
+			},
+		},
+		Configurations: &schema.NVDCVEFeedJSON10DefConfigurations{
+			CVEDataVersion: "4.0",
+			Nodes: []*schema.NVDCVEFeedJSON10DefNode{
+				{
+					CPEMatch: []*schema.NVDCVEFeedJSON10DefCPEMatch{
+						{
+							Cpe23Uri:              `cpe:2.3:a:vmware:spring_cloud_function:*:*:*:*:*:*:*:*`,
```

Comment:
This CPE is a guess for now. I could not find this in NVD

Reply:
https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-function-core

Reply:
```
INFO[0000] wfn:[part=ANY,vendor="pivotal software, inc",product="spring_cloud_function_core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal software, inc",product="spring_cloud_function_core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal software, inc",product="spring",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal software, inc",product="spring",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal software, inc",product="spring\-cloud\-function\-core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal software, inc",product="spring\-cloud\-function\-core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal",product="spring\-cloud\-function\-core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal",product="spring\-cloud\-function\-core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal",product="spring_cloud_function_core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal",product="spring_cloud_function_core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal",product="spring",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="pivotal",product="spring",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="software,",product="spring",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="software,",product="spring",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="software,",product="spring\-cloud\-function\-core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="software,",product="spring\-cloud\-function\-core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="software,",product="spring_cloud_function_core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="software,",product="spring_cloud_function_core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="inc.",product="spring\-cloud\-function\-core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="inc.",product="spring\-cloud\-function\-core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="inc.",product="spring_cloud_function_core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="inc.",product="spring_cloud_function_core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="inc.",product="spring",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="inc.",product="spring",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org",product="spring_cloud_function_core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org",product="spring_cloud_function_core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org",product="spring",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org",product="spring",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org",product="spring\-cloud\-function\-core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org",product="spring\-cloud\-function\-core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="springframework",product="spring\-cloud\-function\-core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="springframework",product="spring\-cloud\-function\-core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="springframework",product="spring_cloud_function_core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="springframework",product="spring_cloud_function_core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="springframework",product="spring",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="springframework",product="spring",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="cloud",product="spring\-cloud\-function\-core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="cloud",product="spring\-cloud\-function\-core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="cloud",product="spring_cloud_function_core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="cloud",product="spring_cloud_function_core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="cloud",product="spring",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="cloud",product="spring",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org.springframework.cloud",product="spring\-cloud\-function\-core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org.springframework.cloud",product="spring\-cloud\-function\-core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org.springframework.cloud",product="spring_cloud_function_core",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org.springframework.cloud",product="spring_cloud_function_core",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org.springframework.cloud",product="spring",version="3.2.2",update=ANY,edition=ANY,language=ANY] 
INFO[0000] wfn:[part=ANY,vendor="org.springframework.cloud",product="spring",version="3\.2\.2",update=ANY,edition=ANY,language=ANY] 
```
Matching products are spring, spring_cloud_function_core, spring-cloud-function-core